apps/vite/src/components/calendar/CalendarGrid.tsx

```
@@ -44,9 +103,17 @@ const CalendarGrid: React.FC = () => {
               )}
-            </div>
-          ))}
+            </button>
+          )
+        })}
       </div>
+      {dayDialogData && (
+        <DayDialog
+          open={dayDialogOpen}
+          setOpen={setDayDialogOpen}
+          children={<DayDetail dayData={dayDialogData} />}
```

Biome

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)